lib/block-supports/typography.php

```diff
@@ -430,19 +431,21 @@ function gutenberg_get_computed_fluid_typography_value( $args = array() ) {
  * @since 6.2.0 Added 'settings.typography.fluid.minFontSize' support.
  * @since 6.3.0 Using layout.wideSize as max viewport width, and logarithmic scale factor to calculate minimum font scale.
  * @since 6.4.0 Added configurable min and max viewport width values to the typography.fluid theme.json schema.
+ * @since 6.5.0 Deprecated bool argument $should_use_fluid_typography.
```

Comment:
Just double-checking, are we aiming to try to get this fix in for 6.5.0, or are we targeting 6.6 now?

Reply:
I reckon 6.6. I was just about to push an update that changes this. Cheers!